twiml/message/message/message-4/message-4.3.x.js

```diff
@@ -1,7 +1,7 @@
 const MessagingResponse = require('twilio').twiml.MessagingResponse;
 
 const response = new MessagingResponse();
-const message = response.message();
+const message = response.message('Hello Jenny');
```

Comment:
OK. By the way, when you were checking for the XML contents to assert matching the sample you are only iterating over the elements and not over the text nodes. This might give false positives.